WordPress/src/main/res/values/strings.xml

```diff
@@ -2869,6 +2869,10 @@
     <string name="premium_upgrades_message">You have active premium upgrades on your site. Please cancel your upgrades prior to deleting your site.</string>
     <string name="show_purchases">Show purchases</string>
     <string name="checking_purchases">Checking purchases</string>
+    <string name="gravatar_info_description">"Gravatar keeps your profile information safe and up to date, automatically syncing any updates made here with your Gravatar profile."</string>
```

Comment:
What are the quotes (") for in this string?

Reply:
Good catch. Removed! Thanks!